modules/candidate_list/README.md

```diff
@@ -0,0 +1,51 @@
+# Candidate List
+
+## Purpose
+
+The candidate list is intended to provide a menu from which users
+can access specific candidates while browsing LORIS data.
+
+## Intended Users
+
+The candidate list is primarily used by data entry staff to use in
+order to access the `timepoint_list` module (and `instrument_list`
+from there) in order to access the candidate and perform data entry.
+
+## Scope
+
+The `candidate_list` module provides a list of candidates, as well
+as linking to other relevant LORIS modules where applicable.
+
+## Permissions
+
+Accessing the `candidate_list` module requires either the `data_entry`
+permission and at least one study site, or the `access_all_profiles`
+permission.
+
+Users with the `access_all_profiles` permission can see every
+candidate in LORIS, while users with only the `data_entry` permission
+can only see candidates at study sites where they are affiliated.
+
+## Configurations
+
+The `useProjects` configuration affects whether the table has a
+"Project" column for the candidate, and whether there is a filter
+dropdown for that column.
+
+The `useEDC` configuration variable has a similar function for the
+"EDC" column.
+
+## Interactions with LORIS
+
+The "Scan Done" column indicates whether or not a scan was performed
+for that candidate. When a scan was performed (i.e. it has the value
```

Comment:
need a coma after i.e.

Reply:
No it doesn't.

Reply:
I have check a few grammar web site and they all state that a comma is needed or at least put one after

Reply:
(fwiw, you're both right.  There's definitely a rule about "i.e.," having a comma, and it's often ignored in informal and technical writing, which is fine by me.)